service/src/main/java/com/codeforcommunity/processor/authenticated/ProtectedUserProcessorImpl.java

```diff
@@ -146,6 +147,19 @@ public void disableUserAccount(JWTData userData, int userId) {
     user.store();
   }
 
+  @Override
+  public void enableUserAccount(JWTData userData, int userId) {
+    if (!userData.isAdmin()) {
+      throw new AdminOnlyRouteException();
+    }
+    UsersRecord user = db.selectFrom(USERS).where(USERS.ID.eq(userId)).fetchOne();
```

Comment:
Also check for users who were not "deleted"

`.where(USERS.DELETED_AT.isNull())`

Reply:
^